From 67610020ce7c321678ef59af95fde0483e8e8149 Mon Sep 17 00:00:00 2001
From: Fabrice Fontaine <fontaine.fabrice@gmail.com>
Date: Sun, 4 Apr 2021 21:09:19 +0200
Subject: [PATCH] lib/pthread-cond.c: use tv.nsec instead of tv.usec

Use tv.nsec instead of tv.usec to avoid the following build failure:

lib/pthread-cond.c: In function 'pthread_cond_wait':
lib/pthread-cond.c:121:16: error: 'struct timespec' has no member named 'tv_usec'; did you mean 'tv_sec'?
  121 |       duration.tv_usec = 0;
      |                ^~~~~~~
      |                tv_sec

Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>
[upstream status: https://lists.gnu.org/archive/html/bug-gnulib/2021-04/msg00044.html]
---
 lib/pthread-cond.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/pthread-cond.c b/lib/pthread-cond.c
index baa7c60bb..c2c00be03 100644
--- a/lib/pthread-cond.c
+++ b/lib/pthread-cond.c
@@ -118,7 +118,7 @@ pthread_cond_wait (pthread_cond_t *cond _GL_UNUSED,
       struct timespec duration;
 
       duration.tv_sec = 86400;
-      duration.tv_usec = 0;
+      duration.tv_nsec = 0;
       nanosleep (&duration, NULL);
     }
 }
-- 
2.30.2

